Integrations/integration-Cisco-umbrella.yml

```diff
@@ -0,0 +1,885 @@
+        content += '\n\n';
+        content += '\n| Name        | Score           |';
+        content += '\n| ------------- |:-------------:|';
+        var relDomains = (coExistingDomains.pfs2) ? coExistingDomains.pfs2 : [];
+        for (var i = 0; i < relDomains.length; i++) {
+            var relatedDomainName = relDomains[i][0];
+            var relatedDomainScore = relDomains[i][1];
+            content += '\n| ' + relatedDomainName + ' | ' + relatedDomainScore + ' |';
+        }
+        var contentResp = formatContent(content);
+        return contentResp;
+    }
+
+
+
+    //////////////////////////
+    // Returns a tableFormatter Markdown Object
+    // Pass in [ { col1 : 'val1', col2 : 1 } , { col1 : 'val2', col2 : 2 } ]
+    function getTableFormatterObj(rowData) {
+        // rows = [ { col1 : 'val1', col2 : 1 } , { col1 : 'val2', col2 : 2 } ]
+
+
+        //return {ContentsFormat: formats.table, Type: entryTypes.note, Contents: rowData};
+        return arrToMd(rowData);
+    }
+
+    function formatTableTitle(tableTitleInfo) {
+        //return {ContentsFormat: 'markdown', Contents: '## ' + tableTitleInfo, Type: 1}
+        return '## ' + tableTitleInfo;
+    }
+
+    function formatContent(markDownContent) {
+        //return {ContentsFormat: 'markdown', Contents: markDownContent, Type: 1}
+        return markDownContent;
+    }
+
+
+    /*
+
+      The following are the API's available URL's for Cisco Investigate
+
+      url: 'https://investigate.api.opendns.com/domains/categorization/<domain>',
+      url: 'https://investigate.api.opendns.com/domains/categorization/ ',
+      url: 'https://investigate.api.opendns.com/domains/categorization/<domain>?showLabels',
+      url: 'https://investigate.api.opendns.com/domains/categories/',
+      url: 'https://investigate.api.opendns.com/recommendations/name/<domain>.json',
+      url: 'https://investigate.api.opendns.com/links/name/<domain>.json',
+      url: 'https://investigate.api.opendns.com/security/name/<domain>.com.json',
+      url: 'https://investigate.api.opendns.com/domains/<domain>/latest_tags',
+      url: 'https://investigate.api.opendns.com/dnsdb/name/a/<domain>.json',
+      url: 'https://investigate.api.opendns.com/dnsdb/ip/a/<ipaddress>.json',
+      url: 'https://investigate.api.opendns.com/ips/<ipaddress>/latest_domains'
+        */
+  type: javascript
+  commands:
+  - name: umbrella-domain-categorization
+    arguments:
+    - name: domain
+      required: true
+      default: true
+      description: Enter the domain you would like to categorize (e.g. amazon.com)
+    outputs:
+    - contextPath: Domain
+      description: Array of domains, each includes Name, ContentCategory & SecurityCategory
```

Comment:
we should have a description for every leaf, e.g. Domain.Name - The domain name, Domain. ContentCategory - The domain security category, etc